packages/utils/src/lib/reports/generate-md-report.ts

```diff
@@ -229,7 +229,8 @@ function reportToAboutSection(
   report: ScoredReport,
   commitData: CommitData | null,
 ): string {
-  const date = new Date().toString();
+  const date = formatDate(new Date());
```

Comment:
Question - do we then want to have the date in two formats or just one? In `core` and `execute-process` we use ISO format (see `collect.ts`, `runner.ts` and relevant tests that use `ISO_STRING_REGEXP`).

Reply:
The human-friendly format should only be used when we're printing something for the user. But ISO is still needed when we need those dates to be parseable later (we send the dates from `report.json` to server) - `toISOString` is perfect for serializing dates so that they will be parsed correctly by pretty much anything, `toLocaleDateString` has no such guarantee.